optimade/server/routers/structures.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends
+from starlette.requests import Request
+
+from optimade.models import (
+    ErrorResponse,
+    StructureResource,
+    StructureResponseMany,
+    StructureResponseOne,
+)
+from optimade.server.config import CONFIG
+from optimade.server.deps import EntryListingQueryParams, SingleEntryQueryParams
+from optimade.server.entry_collections import MongoCollection, client
+from optimade.server.mappers import StructureMapper
+
+from .utils import get_entries, get_single_entry
+
+router = APIRouter()
+
+structures_coll = MongoCollection(
+    client[CONFIG.mongo_database]["structures"], StructureResource, StructureMapper
```

Comment:
I think I'm still in favour of a `CONFIG.structure_collection` that defaults to `"structures"`, but that's very minor.

Reply:
I think this is _very_ back-end specific. We can definitely add a `[MONGO]` section to the config. I also think we shouldn't use the `[DEFAULT]` section, since it's special for `config.ini` and it's fields will always be included in all other sections. This is a problem for the sections specifying provider-specific fields.

Reply:
I get your point, I guess I'm just being lazy by wanting to reuse even `main.py` :P

Reply:
Well, I have actually updated this PR with all your suggested changes - check it out :)